packages/dd-trace/src/telemetry/dependencies.js

```diff
@@ -0,0 +1,25 @@
+const { moduleLoadStart } = require('../channel-itm')
+
+const savedDependencies = []
+const detectedDependencyNames = new Set()
+
+async function start (config, application, host) {
+  const interval = setInterval(() => {
```

Comment:
We should avoid `setInterval` here to avoid idle wake ups. It also seems like there isn't an interval being passed?

Reply:
Yes, I've already changed it in my local by a `setTimeout` that will stop if it hasn't more dependencies to send. this dependencies part is not done here, only in my computer :-) I'll add a commit soon